bliss/surveys/survey.py

```diff
@@ -0,0 +1,76 @@
+import pytorch_lightning as pl
+from torch.utils.data import Dataset
+
+
+class Survey(pl.LightningDataModule, Dataset):
```

Comment:
Do you think it might be good to make this explicitly abstract (inheriting from `ABC`, `@abstractmethod` decorators) to require using an inherited Survey instead of the base class?

Reply:
Great idea! Done.